src/UseCases/ApplyForMembership/MailTemplateValueBuilder.php

```diff
@@ -5,10 +5,13 @@
 namespace WMDE\Fundraising\MembershipContext\UseCases\ApplyForMembership;
 
 use WMDE\Fundraising\MembershipContext\Domain\Model\MembershipApplication;
+use WMDE\Fundraising\MembershipContext\RefactoringException;
 
 class MailTemplateValueBuilder {
 
 	public function buildValuesForTemplate( MembershipApplication $application ): array {
+		// phpcs:disable Squiz.PHP.NonExecutableCode.Unreachable
+		throw new RefactoringException( 'The ApplyForMembershipUseCase needs to pass the Payment type' );
```

Comment:
We'll probably pass the payment (or the display data) as a 2nd param instead of just the payment type (to get to the amount interval and type). But that's for our next-sprint-selves